examples/query-suggestions/package.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "@algolia/query-suggestions",
```

For now, our convention is to keep the `-example` suffix.

```suggestion
  "name": "@algolia/query-suggestions-example",
```

---

name modified.